code/modules/telesci/gps.dm

```diff
@@ -34,6 +34,8 @@ var/list/GPS_list = list()
 /obj/item/device/gps/AltClick(mob/user)
 	if(!in_range(src, user))
 		return
+	if(!user.canUseTopic(user))

```

Comment:
canUseTopic has range checks for all mobs that can return 1 from it.

I dont' know what's going on here, but the previous version should have worked fine and is the *correct* way to do it.

all non living carbon mobs simply return null from canUseTopic, which should evaluate to false.

I'm going to look at this in more detail